src/EditorFeatures/VisualBasicTest/EndConstructGeneration/MethodBlockTests.vb

```diff
@@ -56,10 +47,10 @@ Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.UnitTests.EndConstructGenera
                         "Class c1",
                         "    Sub New()",
                         "",
-                        "        ' This call is required by the designer.",
+                       $"        ' {ThisCallIsRequiredByTheDesigner}",
```

Comment:
It's from VBEditorResources, but the `VBEditorResources.` prefix is unnecessary.